.gitignore

```diff
@@ -0,0 +1,34 @@
+# Prerequisites
```

Comment:
this is a common .gitignore - should be in the repo <root>

.git will not track / upload / download files/folders matched in this file.